Source/JavaScriptCore/runtime/JSGlobalObject.h

```diff
@@ -532,6 +532,7 @@ class JSGlobalObject : public JSSegmentedVariableObject {
     InlineWatchpointSet m_sharedArrayBufferSpeciesWatchpointSet { ClearWatchpoint };
     InlineWatchpointSet m_typedArrayConstructorSpeciesWatchpointSet { IsWatched };
     InlineWatchpointSet m_typedArrayPrototypeIteratorProtocolWatchpointSet { IsWatched };
+    InlineWatchpointSet m_alwaysSlowPutContiguousWatchpointSet { ClearWatchpoint };
```

Comment:
We should give this a more descriptive name. I think it's even ok if it refers to the DOM in its name. But right now, this name isn't describing what's being protected. Might also be worth a comment saying how we can rely on this watchpoint. For example, it's protecting something on the prototype. And it's protecting the "length" getter on the prototype. We still need to protect against self length properties. And we need to ensure self's prototype is the prototype this watchpoint is installed on. Generally, we do that via an original structure check

Reply:
Renamed to `m_alwaysSlowPutContiguousPrototypesAreSaneWatchpointSet` and added a comment on `NodeList.prototype.length`.

We do perform original structure check already, and I'm adding a thorough WPT coverage for that.